src/encoder.rs

```diff
@@ -256,13 +255,11 @@ pub struct DeblockState {
 impl Default for DeblockState {
     fn default() -> Self {
         DeblockState {
-            levels: [0; PLANES+1],
+            levels: [16,16,8,8],
```

Comment:
Can you tell me what each entry of levels[] for? 
I read below lossless checking that both levels[0] > 0 and levels[1] > 0?

Reply:
pub levels: [u8; PLANES+1],  // Y vertical edges, Y horizontal, U, V

Reply:
Oh I see- thanks!